core/storage/classifier/gae_models_test.py

```diff
@@ -17,35 +17,44 @@
 from core.platform import models
 from core.tests import test_utils
 
-(classifier_models,) = models.Registry.import_models([models.NAMES.classifier])
+(classifier_data_models,) = models.Registry.import_models(
+    [models.NAMES.classifier])
 
 
-class ClassifierModelUnitTests(test_utils.GenericTestBase):
-    """Test the ClassifierModel class."""
+class ClassifierDataModelUnitTests(test_utils.GenericTestBase):
+    """Test the ClassifierDataModel class."""
 
     def setUp(self):
-        super(ClassifierModelUnitTests, self).setUp()
-        classifier_models.ClassifierModel.create('exp_id1', 1, 'state_name1',
-                                                 'LDAStringClassifier',
-                                                 {'alpha': 1.0}, 1)
-        classifier_models.ClassifierModel.create('exp_id1', 1, 'state_name2',
-                                                 'LDAStringClassifier',
-                                                 {'alpha': 1.0}, 1)
-        classifier_models.ClassifierModel.create('exp_id2', 1, 'state_name3',
-                                                 'LDAStringClassifier',
-                                                 {'alpha': 1.0}, 1)
+        super(ClassifierDataModelUnitTests, self).setUp()
+        classifier_data_models.ClassifierDataModel.create('job_request_id1',
```

Comment:
you can move this to the next line to pull the subsequent lines on a single line.